src/runtime_src/core/edge/drm/zocl/sched_exec.c

```diff
@@ -662,6 +662,26 @@ init_cus(struct sched_cmd *cmd)
 		DRM_INFO("CU can only be initialized once.\n");
 }
 
+/*
+ * returns false if any of the cu doesnt support interrupt
+ */
+bool
+get_cus_support_intr(struct drm_zocl_dev *zdev)
+{
+	struct ip_data *ip;
+        int i;
+        if (!zdev->ip)
+                return false;
+
+        for (i = 0; i < zdev->ip->m_count; ++i) {
+                ip = &zdev->ip->m_ip_data[i];
+                if(!(ip->properties & 0x1)) {
```

Comment:
Confirmed with system compiler, if the CU has a interrupt pin, the bit 0 of properties will be set to 1. This is the correct way to determine if a CU supports interrupt.